### DISCUSSION TITLE
Some models not being loaded in quicklook

### DISCUSSION THREAD

**Discussion by mr339** [google/model-viewer#2208] [Q&A]
### Description
 I've started getting this issue where some models don't load on quicklook with a message saying "Object requires a new version of IOS" even though the device has the latest version available i.e 13.5.1.It's working fine on android devices,
- [10] Android
- [83.0.4103.101 ] Chrome

This issue didn't occur until the newest version of model viewer was released or it did exist in v0.10.0 and i hadn't tested the particular models that are giving the issues right now.

### Browser and OSAffected
- [13] Safari
- [13.5.1 ] iOS

### Versions
- model-viewer: v1.0.0

this is the screen i get
[https://pasteboard.co/JcHdx0M.png](url)

**Comment by elalish** [comment]
There's no connection between Android and iOS in this case, since they use different models (glTF for Android/web, and usdz for iOS). So it can only be a problem with the usdz model; perhaps it needs to be reconverted?

**Comment by wedaren** [comment]
I use apple usdzconvert transform from gltf to usdz, usdz file open in ios is ok, but in moder-viewer is error  with a message saying "Object requires a new version of IOS"

**Comment by elalish** [comment]
@wedaren That's interesting; can you ask on Apple's forums? If you get an answer it would be great if you could post it here as well, because I'd like to know. model-viewer is just activating QuickLook with your file URL, so we really don't have any control over this.

**Comment by wedaren** [comment]
I am poor in English. This link maybe help. https://developer.apple.com/forums/thread/120523

**Comment by elalish** [comment]
Ah, thanks, that makes sense. Since quick-look is a separate app from Safari, it makes sense it wouldn't have the same authentication.

**Comment by WayneEld** [comment]
People also experiencing the same issue in Quick Look just on iOS. [https://developer.apple.com/forums/thread/656143 ](url).

It may be a usdz issue.

**Comment by jeffscottward** [comment]
Just hit this issue with this file
https://onedrive.live.com/?authkey=%21APFor0C9phSoVBc&cid=AA8A24AA7E2F56BF&id=AA8A24AA7E2F56BF%2124950&parId=AA8A24AA7E2F56BF%2124786&o=OneUp

**Comment by jenhausu** [comment]
Check is your object's name is typed correctly, or have a space in front of the object's name. it works for me.

**Comment by elalish** [comment]
@grorg Another thread that may be of interest regarding Quick Look issues.
